modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -37,19 +38,17 @@ export interface ModeRenderInfo {
 }
 
 export interface RenderInfo {
-  readonly getMothership: () => Gui.GuiSystem;
-  readonly getUiMothership: () => Gui.GuiSystem;
-  readonly backstage: Backstage.UiFactoryBackstage;
+  readonly forPopups: {
```

Comment:
Is there a reason why we aren't consistent with the naming here? Everywhere else we use `popups` and `dialogs`

Reply:
Definitely no reason.

- [ ] fix consistency with naming.